bundles/admin/admin-layereditor/instance.js

```diff
@@ -1,4 +1,7 @@
 import { LayerEditorFlyout } from './view/Flyout';
+import { OpenLayerEditorRequest } from './request/OpenLayerEditorRequest';
```

Comment:
I would rename this Open -> Show just to make it less confusing reg OpenLayers :)